src/networking.c

```diff
@@ -287,8 +287,12 @@ int prepareClientToWrite(client *c) {
     /* If CLIENT_CLOSE_ASAP flag is set, we need not write anything. */
     if (c->flags & CLIENT_CLOSE_ASAP) return C_ERR;
 
-    /* CLIENT REPLY OFF / SKIP handling: don't send replies. */
-    if (c->flags & (CLIENT_REPLY_OFF|CLIENT_REPLY_SKIP)) return C_ERR;
+    if (c->flags & CLIENT_PUSHING) {
+        /* PUSH messages are not really a reply, and should not be skipped. */
+    } else {
+        /* CLIENT REPLY OFF / SKIP handling: don't send replies. */
+        if (c->flags & (CLIENT_REPLY_OFF|CLIENT_REPLY_SKIP)) return C_ERR;
+    }
```

Comment:
In fact, the change to the next line is only indentation, and the git blame side will not be affect
that's why I did it, i don't want to change the git blame log. i can change it if this needed

Reply:
git blame (and CR diff, future merge conflicts) do get hit when indentation changes. or maybe i'm missing something?
anyway, in this case i think it's minor (just two lines), and i think we can afford to write this nicer.

Reply:
you are right, i guess that because i am using -w option by default (IDE CLion), see https://git-scm.com/docs/git-blame#Documentation/git-blame.txt--w